video.py

```diff
@@ -85,26 +93,50 @@ def process_frames(original_filename, output_filename):
 
     cv2.destroyAllWindows()
     video.release()
+    video_mask.release()
+
+    # encode videos to h264
+    video_enc_proc = start_encode_video(output_filename + '.lossless.mkv', output_filename)
+    video_mask_enc_proc = start_encode_video(mask_filename + '.lossless.mkv', mask_filename)
+    assert video_enc_proc.wait() == 0, 'Video encoding failed'
+    assert video_mask_enc_proc.wait() == 0, 'Mask video encoding failed'
+
+def start_encode_video(src, dst):
+    return subprocess.Popen([
+        imageio_ffmpeg._utils.get_ffmpeg_exe(),
+        '-y', # overwrite output
+        '-i', src, # input
+        '-codec', 'h264', # encode using x264
+        '-crf', '20', # aims for consistent video quality of 20, using as much bitrate as needed
+        dst
+    ])
 
 # creates a video with src_video as a video source and src_audio as an audio source
 
 
 def insert_audio(src_audio, src_video, dst):
-    src_video = VideoFileClip(src_video)
-    src_audio = VideoFileClip(src_audio)
-    audio = src_audio.audio
-    final_video = src_video.set_audio(audio)
-    final_video.write_videofile(dst)
+    subprocess.check_call([
```

Comment:
@axmichael 
did you change the encoding pipeline for better performances?

Reply:
Yes. This change avoids encoding the video a second time, and also keeps the original audio instead of re-encoding it.